src/pages/Calculator/Steps/SeedingMethod/index.js

```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
                 {council === 'MCCC'
                   && (
                   <>
-                    {renderMethod('Precision', methods[seed.label]?.Precision)}
-                    {renderMethod('Drilled', methods[seed.label]?.Drilled)}
+                    {renderMethod('Precision', methods[seed.label]?.Precision, '', 'Precision')}
+                    {renderMethod('Drilled', methods[seed.label]?.Drilled, '', 'Drilled')}
```

Comment:
why did you add another param with the same value? and why another param with empty string?